packages/components/src/navigation/NavTabList.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import NavTabList, { type NavTabItem } from './NavTabList';
+
+// Helper to build tabs
+function makeTabs(count = 3): NavTabItem[] {
+  return Array.from({ length: count }, (_, i) => ({
+    key: `TAB_${i + 1}`,
+    title: `Tab ${i + 1}`,
+    isClosable: false,
+  }));
+}
+
+// JSDOM doesn't implement scrollIntoView; stub to avoid errors triggered by effect
+beforeAll(() => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (window.HTMLElement as any).prototype.scrollIntoView = jest.fn();
+});
```

Comment:
You can just use `window.HTMLElement.prototype.scrollIntoView = jest.fn()` which we use in another test without the `beforeAll` or any cast.